openweather.js

```diff
@@ -273,6 +287,7 @@ function prepareUpdate(forecast, weather, full) {
             buildDeltaUpdate(simpleWeatherCode, weather.weathercode)
         ];
         case 'meta-simple': return [
+            buildDeltaUpdate(currentTempPath, latest.currentTemp),
```

Comment:
Value is wrong for meta-data, needs hold units, ie. `{ units: "K"} `

Reply:
fixed, thank you